cli/tests/pcluster/validators/utils.py

```diff
@@ -26,6 +26,9 @@ def assert_failure_messages(actual_failures, expected_messages):
             res = any(re.search(expected_message, actual_failure.message) for actual_failure in actual_failures) or any(
                 expected_message == actual_failure.message for actual_failure in actual_failures
             )
+            # Only printed to console output if there is an assertion error
+            print(f"Expected Message: {expected_message}")
+            print(f"Actual Failures: {actual_failures}")
```

Comment:
Wouldn't this print the `expected_message` and the `actual_failures` all the time?

Reply:
OK, let's say explicitly in the comment that this is a feature of PyTest.